python/src/deltachat/testplugin.py

```diff
@@ -373,6 +369,13 @@ def wait_configure_and_start_io(self):
             for acc in started_accounts:
                 acc._evtracker.wait_all_initial_fetches()
 
+        def wait_configure(self, acc):
```

Comment:
But which advantage would this have? I mean, of course we could write `AccountMaker.wait_configure()` instead of `acfactory.wait_configure()` but currently we are not even importing `AccountMaker` in `test_account.py`. And I'd